src/coreclr/jit/fgehopt.cpp

```diff
@@ -1232,13 +1230,6 @@ PhaseStatus Compiler::fgCloneFinally()
             continue;
         }
 
-        // Skip cloning if the finally is rarely run code.
-        if (isAllRare)
-        {
-            JITDUMP("Finally in EH#%u is run rarely; skipping.\n", XTnum);
-            continue;
-        }
-
         // Empirical studies from CoreCLR and CoreFX show that less
```

Comment:
Unrelated to this PR, but it might be interesting to revisit comments like these by looking at some JIT metrics.

Reply:
Yes, that is on my todo list ...